dr_spaam_ros2:
  ros__parameters:
    weight_file: "/home/hyz/Project/ros_ws/install/dr_spaam_ros2/share/dr_spaam_ros2/model_weight/dr_spaam_e40.pth"
    detector_model: "DR-SPAAM"  # DROW3 or DR-SPAAM
    conf_thresh: 0.95
    stride: 5  # use this to skip laser points
    panoramic_scan: True  # Set to true if the scan covers 360 degrees
    use_gpu: False  # Set to True if CUDA is available, False for CPU-only

